bve/src/parse/mesh/instructions/execution.rs

```diff
@@ -156,6 +158,7 @@ impl Executable for CreateMeshBuilder {
 
             let first: &PolygonFace = group.peek().expect("Groups must not be empty");
             // All of these attributes are the same, so we can just grab them from the first one.
+            // Yay duplicate data
```

Comment:
I need to go back on a conversation with my mod to see if this is actually duplicate data in the spec or if I'm reading the spec wrong, iirc I was reading the spec wrong and it was impossible for it to be different so I should just store this on a per mesh (as opposed to a pet polygon) basis.

Reply:
Not solved, but going to resolve later on the TODO.